Automattic/jetpack
# Protect: Add truncation in `NavigationGroup`

Insert truncate logic at the `NavigationGroup` component, which will truncate if has more than 8 items.

#### Changes proposed in this Pull Request:

* Add truncate handling inside `NavigationGroup`.

#### Jetpack product discussion


1202064326332560-as-1202183900595761/f

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `Protect`
* Navigate to the admin page.
* You should see it truncating if has more than 8 items.


#### Demo

https://user-images.githubusercontent.com/1663717/165559170-f7419f55-174e-469c-b102-952a490d4748.mp4
